src/SearchCriteria/SearchCriteriaBuilder.php

```diff
@@ -80,11 +72,8 @@ protected static function propertiesMap()
     protected static function extractPropertyFromArray(string $propertyName, array $data, $default = '')
     {
         $map = self::propertiesMap();
-        if (!isset($map[$propertyName])) {
```

Comment:
Discussed the removal of this bit. @cleancode4ever pointed out that it was overly paranoid, and that with the way the system is formulated he couldn't actually test it meaning there wasn't effectively a way to ever invoke it.